tests/tkupchyn/homework_08/test_three_chefs_alt.py

```diff
@@ -0,0 +1,27 @@
+import pytest
+from tkupchyn.homework_08.three_chefs_alt import (AbstractCook, ItalianCook, UkrainianCook, JapaneseCook)
+
+
+@pytest.mark.parametrize("cook, orders, expected_result",
+                         [(AbstractCook,
+                           [('Food', 2, 13), ('fooD', 5), ('drink', 4)],
+                           "Food: 151, Drink: 124, Total:275"),
```

Comment:
"Food: 151, Drink: 124, Total:275"    for total no space after semicolumn
"Food: 151, Drink: 124, Total: 275"